tests/api-mongo.test.js

```diff
@@ -1,4 +1,9 @@
 import chai from 'chai';
+<<<<<<< HEAD
```

Comment:
Seems like you have a lot of unaddressed merge conflicts.

This guide should help - https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/addressing-merge-conflicts